spotless.gradle

```diff
@@ -10,7 +10,7 @@ spotless {
     }
     kotlin {
         target '**/*.kt'
-        ktlint('0.39.0').userData(['disabled_rules': 'no-wildcard-imports, import-ordering, no-blank-line-before-rbrace'])
+        ktlint('0.42.0').userData(['disabled_rules': 'no-wildcard-imports, import-ordering, no-blank-line-before-rbrace'])
```

Comment:
Ktlint needed upgraded because of https://github.com/pinterest/ktlint/issues/1160